Command/RequestAPICommand.php

```diff
@@ -14,24 +14,29 @@ protected function configure()
     {
         $this
             ->setName('ac:request-api')
-            ->addArgument('client-key', InputArgument::REQUIRED)
-            ->addArgument('rest-url', InputArgument::REQUIRED)
-            ->setDescription('Request REST end-points');
+            ->addArgument('client-key', InputArgument::REQUIRED,"Client-key from tenant OR tenant id macro, like t#123")
+            ->addArgument('rest-url', InputArgument::REQUIRED, "REST api endpoint, like /rest/api/2/issue/{issueIdOrKey}")
+            ->setDescription('Request REST end-points. 
+Documentation available on https://docs.atlassian.com/jira/REST/cloud/');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $tenant = $input->getArgument('client-key');
         $restUrl = $input->getArgument('rest-url');
         $em = $this->getContainer()->get('doctrine')->getManager();
-        $tenant = $em->getRepository('AtlassianConnectBundle:Tenant')
-            ->findOneByClientKey($tenant);
+        if(preg_match("'^t\#(\d+)$'",$input->getArgument('client-key'),$m)) {
```

Comment:
@thecatontheflat ok, added in another PR - https://github.com/thecatontheflat/atlassian-connect-bundle/pull/13

please close this one (its from master branch, cant create other PR)